ui/src/views/tenants/CreateTenantUser.tsx

```diff
@@ -1,27 +1,79 @@
 import React, { Component } from "react";
+import { notification } from "antd";
 import { Link, RouteComponentProps } from "react-router-dom";
 
 import { Space, Breadcrumb, Card, PageHeader } from "antd";
 
-import { Tenant, TenantUser, AddTenantUserRequest } from "@chirpstack/chirpstack-api-grpc-web/api/tenant_pb";
+import { Tenant, TenantUser } from "@chirpstack/chirpstack-api-grpc-web/api/tenant_pb";
+import { User } from "@chirpstack/chirpstack-api-grpc-web/api/user_pb";
 
 import TenantUserForm from "./TenantUserForm";
-import TenantStore from "../../stores/TenantStore";
+
+// stores
+import SessionStore from "../../stores/SessionStore";
 
 interface IProps extends RouteComponentProps {
   tenant: Tenant;
 }
 
-class CreateTenantUser extends Component<IProps> {
-  onFinish = (obj: TenantUser) => {
-    obj.setTenantId(this.props.tenant.getId());
+interface IState {
+  user?: User;
+}
+
+class CreateTenantUser extends Component<IProps, IState> {
+  constructor(props: IProps) {
+    super(props);
+
+    this.state = {
+      user: undefined,
+    };
+  }
 
-    let req = new AddTenantUserRequest();
-    req.setTenantUser(obj);
+  componentDidMount() {
+    SessionStore.on("change", () => {
+      this.setState({
+        user: SessionStore.getUser(),
+      });
+    });
 
-    TenantStore.addUser(req, () => {
-      this.props.history.push(`/tenants/${this.props.tenant.getId()}/users`);
+    this.setState({
+      user: SessionStore.getUser(),
     });
+  }
+
+  onFinish = async (obj: TenantUser) => {
+    // Highjack the request to login server
+    if (this.state.user) {
+      const res = await fetch("http://localhost:4000/api/invite_user", {
```

Comment:
change localhost:4000 to console_login_server:4000 for the docker-compose usage in CLS